testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/policy/PasswordPolicyTest.java

```diff
@@ -75,7 +79,7 @@ public void testDigits() {
             RealmModel realmModel = session.getContext().getRealm();
             PasswordPolicyManagerProvider policyManager = session.getProvider(PasswordPolicyManagerProvider.class);
 
-            realmModel.setPasswordPolicy(PasswordPolicy.parse(session, "digits"));
+            realmModel.setPasswordPolicy(PasswordPolicy.parse(session, DigitsPasswordPolicyProviderFactory.ID));
```

Comment:
Please remove this. Tests are by intent using a string and not a constant here.

Reply:
done